tests/AspNetCore.Lambda.SampleApp/project.json

```diff
@@ -6,9 +6,12 @@
     "compilerName": "fsc",
     "compile": {
       "includeFiles": [
+        "Person.fs",        
         "Program.fs"
       ]
-    }
+    },
+    "embed": [ "**/*.cshtml" ],
```

Comment:
When using the physical RazorLight engine then the views don't need to be embedded and this line can be removed.